### ISSUE TITLE
Excel - Listing formulas or notes in element list fails if selection in the current worksheet covers multiple cells

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#20806]
When I select multiple cells in an Excel sheet and open the elements list with `NVDA+F7`, NVDA cannot list the formulas or notes in the sheets.

I think this had already been discussed in another issue but cannot find it. Feel free to close as duplicate if relevant.

Below is an STR with formulas, but the same issue occurs with notes.

### Steps to reproduce
* Open an Excel document with at least one formula, or create one.
* With `shift+arrows`, select at least 2 cells
* Press NVDA+F7
* Select Formulas radio button
### Actual behavior
Listing formulas in the sheets fails
### Expected behavior
Formulas of the current sheet should be listed in the dialog box.
### NVDA logs, crash dumps and other attachments
ERROR - queueHandler.flushQueue (16:17:21.273) - MainThread (5796):
```
Error in func ElementsListDialog.initElementType
Traceback (most recent call last):
  File "queueHandler.pyc", line 67, in flushQueue
  File "browseMode.pyc", line 1576, in initElementType
  File "NVDAObjects\window\excel.pyc", line 714, in _iterNodesByType
  File "NVDAObjects\window\excel.pyc", line 1559, in __init__
AttributeError: 'ExcelSelection' object has no attribute 'excelCellInfo'
```
#### NVDA installed/portable/running from source
Installed
#### NVDA version
2026.3beta1
#### Windows version
Windows 11 23H2 (AMD64) build 22631.7517

#### Name and version of other software in use when reproducing the issue
Microsoft® Excel® LTSC MSO (16.0.14334.20848) 64 bits
#### Other information about your system
N/A
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors
Not re-tested, but I remember that this issue was already present on older NVDA versions, even before 2026.1
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the System Accessibility Repair Tool in NVDA's tools menu?
Not tested, but installed 2026.3beta1 today; should have been run during install.
